### DISCUSSION TITLE
Do handlebars planners have access to set Context?

### DISCUSSION THREAD

**Discussion by DLTKTimClamp** [microsoft/semantic-kernel#4403] [Q&A]
In previous builds the SequentialPlanner had access to set values on the context object. This was helpful when testing a planner as you could test the results of an individual step within the planner. Transitioning to the handlebar planner I have access to set variables that are accessible to the steps, however those variables are not available on the context.

**Comment by markwallace-microsoft** [comment]
I am cleaning up old unanswered discussions
